massdriver-application-gcp-vm/providers.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_providers {
+    jq = {
+      source = "massdriver-cloud/jq"
+    }
+
+    massdriver = {
+      source = "massdriver-cloud/massdriver"
+    }
+
+    mdxc = {
+      source = "massdriver-cloud/mdxc"
+    }
+
+    google = {
+      source = "hashicorp/google"
+    }
+
+    google-beta = {
+      source = "hashicorp/google-beta"
+    }
+
+    null = {
```

Comment:
Are we using null resources?

Reply:
Yes, the alarm channel module uses a null resource to check / turn on APIs.